Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -13,11 +14,25 @@ namespace Doppler.ContactPolicies.Api.Controllers
     [ApiController]
     public class ContactPolicies : ControllerBase
     {
-        [Authorize(DopplerSecurity.Policies.OWN_RESOURCE_OR_SUPERUSER)]
+        private readonly IContactPoliciesService _contactPoliciesService;
+
+        public ContactPolicies(IContactPoliciesService contactPoliciesService)
+        {
+            _contactPoliciesService = contactPoliciesService;
+        }
+
+        [Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
         [HttpGet("/accounts/{accountName}/settings")]
-        public string GetContactPoliciesSettings(string accountName)
+        public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
         {
-            return $"Hello! \"you\" that have access to the account with accountName '{accountName}'";
+            var contactPoliciesSettings = await _contactPoliciesService.GetContactPoliciesSettingsAsync(accountName);
+
+            if (contactPoliciesSettings == null)
+            {
+                return NotFound($"No contact policies settings for account name: {accountName}");
```

Comment:
What do you think about this message?

```suggestion
                return NotFound($"Account {accountName} does not exist.");
```

I mean, any existent user has _contact policies settings_.

Reply:
ok